documentation/assemblies/assembly-deployment-configuration-kafka-mirror-maker.adoc

```diff
@@ -11,30 +11,59 @@
 :parent-context-deployment-configuration-kafka-mirror-maker: {context}
 
 [id='assembly-deployment-configuration-kafka-mirror-maker-{context}']
-= Kafka Mirror Maker configuration
+= Kafka MirrorMaker configuration
 
 :context: deployment-configuration-kafka-mirror-maker
 
-This chapter describes how to configure the `KafkaMirrorMaker` resource to support a Kafka Mirror Maker deployment in your cluster.
+This chapter describes how to configure a Kafka MirrorMaker deployment in your cluster.
 
-The following procedures show how the resource is configured:
+You can use {ProductName} with MirrorMaker or {MirrorMaker}.
```

Comment:
Seems a bit confusing for potential contributors. Why not use `{MirrorMaker2}`?

Reply:
At this point, you could explain that Mirror Maker 2.0 is a more efficient way to mirror data between Kafka clusters, and that it was introduced in Kafka 2.4.0.

Reply:
I've replaced the variable to avoid any confusion. 
I'll say that MirrorMaker 2.0 is the later version and is, essentially, better, but I'll avoid any version numbers.